scrap_amazon.py

```diff
@@ -0,0 +1,61 @@
+import bs4
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import urllib
+import time
+
+def display_data(Products):
+
+	for i in allProducts:
+		detailProduct=i.find("div", {"class": "sg-col-inner"})
+		print(detailProduct.text)
+		print("********************") 
+
+	print(driver.title)
+
+if __name__ == '__main__':
+	
+	driver=webdriver.Chrome(executable_path="E:\chromedriver.exe")
+	my_url = "https://www.amazon.com"
+	driver.get(my_url)
+
+	driver.maximize_window()
+	time.sleep(2)
+	
+	product = input("ENTER PRODUCT : ").strip()

```

Comment:
**MAJOR**
Try taking this from command line input.
`python scrap_amazon.py --product PRODUCT`
You can use the same `click` functionality here also.

Reply:
Will be added.

Reply:
Add this functionality before pushing to `master`